packages/odf/features.ts

```diff
@@ -150,9 +150,10 @@ export const detectRGW: FeatureDetector = async (setFlag: SetFeatureFlag) => {
   id = setInterval(logicHandler, 15 * SECOND);
 };
 
-// ToDo: Add logic to detect ROSA environment here
-export const detectROSA: FeatureDetector = async (setFlag: SetFeatureFlag) =>
-  setFlag(ROSA_FLAG, false);
+export const detectROSA: FeatureDetector = async (setFlag: SetFeatureFlag) => {
+  const isROSA = (window as any).SERVER_FLAGS.branding === 'rosa';
```

Comment:
```suggestion
  const isROSA = (window as WindowObject).SERVER_FLAGS.branding === 'rosa';
```
ex: https://github.com/red-hat-storage/odf-console/blob/master/packages/shared/src/constants/prometheus.ts#L4